streaming/src/test/org/apache/hive/streaming/TestStreaming.java

```diff
@@ -1019,8 +1019,8 @@ private void checkDataWritten2(Path partitionPath, long minTxn, long maxTxn, int
         min = pd.getMinWriteId();
       }
     }
-    Assert.assertEquals(minTxn, min);
-    Assert.assertEquals(maxTxn, max);
+    Assert.assertEquals(minTxn + 1, min);
```

Comment:
I think it will be helpful to add a comment on why there is a +1 otherwise it may not be very easy to understand just by reading the test cas.e

Reply:
Added a comment